compiler/quilt/test/build_no_contents_node.yml

```diff
@@ -0,0 +1,2 @@
+does_not:
+  contain: contents.node
```

Comment:
for proper test coverage we should have a build file without 'contents', a `quilt.yml` file without 'package'

Reply:
`packages` exception tests in `test_command.py:test_install_dependencies_from_invalid_file` where temporary `.yml` created.